src/DayDifference.php

```diff
@@ -2,8 +2,6 @@
 
 namespace johncorrelli;
 
-use DateTimeInterface;
-
```

Comment:
Gah, I don't love that this rule is defaulted. We can leave it since it's such a small repo.